code/API_definitions/qos-booking-and-assignment.yaml

```
@@ -276,7 +275,7 @@ paths:
                 BOOKING_CANCELLED:
                   $ref: "#/components/examples/BOOKING_CANCELLED"
         "202":
-          description: Deletion request accepted to be processed. It applies for an async deletion process. `status` in the response will be `SCHEDULED`
+          description: Deletion request accepted to be processed. It applies for an async deletion process. `status` in the response will be `SUCCESSFUL` and `statusInfo` will be `DELETE_REQUESTED`

```

```suggestion
          description: Deletion request accepted to be processed. It applies for an async deletion process. `status` in the response will be `PENDING` and `statusInfo` will be `DELETE_REQUESTED`
```

The example CANCELLATION_PENDING has the correct status `PENDING`.